lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -348,6 +354,21 @@ def add_check_manifest_lock_script_phase
         # @!group Private Helpers
         #---------------------------------------------------------------------#
 
+        # @return [String] Extension in the target that corresponds to the
+        #         input extension.
+        #
+        def output_extension_for_input_extension(input_extension)
```

Comment:
this method name is a bit generic although it is only used for resource extensions.

Most probably rename to `output_extension_for_resource`

Reply:
and update the documentation above